tests/bluechi_test/systemctl.py

```diff
@@ -27,6 +27,15 @@ def _do_operation_on_unit(self, unit_name: str, operation: str, check_result: bo
                 f"Failed to {operation} service {unit_name} with {result} (expected {expected_result}): {output}")
         return result, output
 
+    def _do_operation(self, operation: str, check_result: bool, expected_result: int) \
+            -> Tuple[Optional[int], Union[Iterator[bytes], Any, Tuple[bytes, bytes]]]:
+
+        result, output = self.client.exec_run(f"{SystemCtl.binary_name} {operation}")
```

Comment:
Hmm, I forgot about this. Debug logging is already included in exec_run: https://github.com/eclipse-bluechi/bluechi/blob/main/tests/bluechi_test/client.py#L82

So I'd rather post another commit to remove debug logging from _do_operation_on_unit to prevent repeating info in logs. WDYT?

Reply:
Oh, good point! Yes, it would be better to remove it from `_do_operation_on_unit` - but only the second one (https://github.com/eclipse-bluechi/bluechi/blob/main/tests/bluechi_test/systemctl.py#L24). The first one is still useful, I think, and no duplication - plus it gets logged always (provided the debug log level).

Reply:
But current logging in exec_run is always printed in debug, `container.exec_run` is not raising exceptions,
so now for each `exec_run` invocation we have following:

```
2024-02-26 14:42:30+0100,847 DEBUG   [bluechi_test.client] Executed command 'systemctl start bluechi-controller' with result '0' and output 'b''' (client:82)```

So is there a value to have following 2 lines instead of 1?

```
2024-02-26 14:42:30+0100,656 DEBUG   [bluechi_test.systemctl] start unit 'bluechi-controller' (systemctl:22)                                                                                                                    
2024-02-26 14:42:30+0100,847 DEBUG   [bluechi_test.client] Executed command 'systemctl start bluechi-controller' with result '0' and output 'b''' (client:82)
```

Reply:
Hmm... you are right. The debug log in systemctl could also be removed.